core/views.py

```diff
@@ -92,3 +92,20 @@ def activity(request, activityId):
             activity.delete()
         return redirect('activities')
     return render(request, template_name, locals())
+
+
+def register_user(request):
+    template_name = 'register.html'
+    if request.method == 'POST':
+        user = request.POST['user']
+        email = request.POST['email']
+        senha = request.POST['senha']
+        newUser = User.objects.create_user(username=user, email=email, password=senha)
+        newUser.save()
```

Comment:
A tabela de estudante nao ta sendo salva nada, isso é ruim pq na hora que tu for pegar as notas dos alunos vai dar trabalho.

Reply:
eu estava pensando nisso, no caso tenho que salvar o nome do estudante na tabela estudante?

Reply:
humrum